### PR TITLE
Added new model for mapping the results returned by both the KfsDetai…

### PR DESCRIPTION
**Pull request by taylorkj**
…ls and KfsHoldingDetails controllers.
@srkirkland @jpknoll @jSylvestre 

This class will allow the return of a strongly typed list of objects for both of the aforementioned controllers.

### CODE REVIEW

File: GiftModels/KfsDetails.cs [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        public string DocNumber { get; set; }
+
+        /// <summary>
+        /// The line item number associated with the KFS transaction used to distinguish it from any other transaction submitted on the same document.  
+        /// This will be unique per DocOriginCode and DocNumber within a scrubber file.  
+        /// The DocOriginCode, DocNumber and SequenceNumber uniquely identify a KFS transaction across all time.
+        /// Note that there can be multiple document number contained within a single scrubber file, so the SequenceNumber by itself may not be unique.  
+        /// It is only unique in combination with its corresponding DocOriginCode and DocNumber.
+        /// </summary>
+        public string SequenceNum { get; set; }
```
Comment (jpknoll) [inline GiftModels/KfsDetails.cs]: @taylorkj For the common models, I've preferred the full names. So in this case, SequenceNumber. You may have to change your sql statement.

File: GiftModels/KfsDetails.cs [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+        /// <summary>
+        /// The eight (8) character Organization Reference ID associated with the KFS transaction.
+        /// </summary>
+        public string OrgReferenceId { get; set; }
+
+        /// <summary>
+        /// The date the associated KFS transaction posted to the general ledger.
+        /// </summary>
+        public DateTime GlPostedDate { get; set; }
```
Comment (jpknoll) [inline GiftModels/KfsDetails.cs]: @taylorkj Maybe shorten this to just PostedDate?

**Comment by jpknoll** [comment]
@taylorkj Otherwise, looks good. :octopus:

File: GiftModels/KfsDetails.cs (hunk 1 above)
Comment (taylorkj) [inline GiftModels/KfsDetails.cs]: This may cause an issue with Jason's logic as he's expecting the abbreviated name.

Comment (jpknoll) [inline GiftModels/KfsDetails.cs]: @jSylvestre Is this an easy fix?

**Comment by taylorkj** [comment]
@jSylvestre :  Are you OK with the above changes that John has suggested?  This will mean you will need to alter any reports using results returned from the KfsDetails and KfsHoldingDetails controllers.

File: GiftModels/KfsDetails.cs (hunk 1 above)
Comment (jSylvestre) [inline GiftModels/KfsDetails.cs]: @jpknoll Don't know. Looking into a Commencement issue at the moment

Comment (srkirkland) [inline GiftModels/KfsDetails.cs]: @jSylvestre @jpknoll I assume we'd have to grab the latest giftModels in both the service and the app and fix any compilation errors.  Also the kfsPreview popup would probably be off because the JSModels would need to be updated to match along with the handlebar bindings.

Comment (taylorkj) [inline GiftModels/KfsDetails.cs]: @jpknoll @srkirkland @jSylvestre 
How about I just leave the names as-is for now and just merge the branch?  That way nothing should break.  We can always update the names at a later date.

Comment (srkirkland) [inline GiftModels/KfsDetails.cs]: @taylorkj deal ;)

**Comment by srkirkland** [comment]
@taylorkj when you are done can you update the giving service with the new packages and then push to dev AND prod?  I'd like to update the prod great app too
